R/compat-oldrel.R

```diff
@@ -0,0 +1,25 @@
+
+# This file serves as a reference for compatibility functions for old
+# versions of R.
+
+
+# R 3.2.0 ------------------------------------------------------------
+
+if (utils::packageVersion("base") < "3.2.0") {
+
+  dir_exists <- function(path) {
+    !identical(path, "") && file.exists(paste0(path, .Platform$file.sep))
+  }
+  dir.exists <- function(paths) {
+    vapply_lgl(paths, dir_exists)
+  }
+
+  names <- function(x) {
```

Comment:
I didn't know that the previous code didn't work, I just wasn't sure and it might've been ok.  So as long as the tests still pass, I'm ok.